spec/controllers/spree/adyen_redirect_controller_spec.rb

```diff
@@ -102,8 +102,8 @@ def params
           order.user_id = 1
           controller.stub(current_order: order)
 
-          ActionController::TestRequest.any_instance.stub(:ip).and_return("127.0.0.1")
-          ActionController::TestRequest.any_instance.stub_chain(:headers, env: env)
+          allow_any_instance_of(ActionController::TestRequest).to receive(:ip).and_return('127.0.0.1')
+          allow_any_instance_of(ActionController::TestRequest).to receive_message_chain(:headers, :env).and_return(env)
         end
 
         it "redirects user to confirm step" do
```

Comment:
We've updated all of the specs rather than just those for the feature we're using (the `AdyenPaymentEncrypted` gateway) as the aim of this pull request is just to update the Adyen gem keeping as much in tact as possible